.github/workflows/pipelines-delegated.yml

```diff
@@ -8,20 +8,34 @@ on:
 
 env:
   PIPELINES_CLI_VERSION: v0.9.8
+  PIPELINES_ACTIONS_VERSION: main
 
 jobs:
   detect_changes:
     name: Detect Infrastructure Changes
     runs-on: ubuntu-latest
     steps:
+      - name: Checkout Pipelines Actions
+        uses: actions/checkout@v4
+        with:
+          path: pipelines-actions
+          repository: gruntwork-test/pipelines-actions
+          ref: ${{ env.PIPELINES_ACTIONS_VERSION }}
```

Comment:
Perhaps now we can use the CODE_ACCESS_TOKEN and make the repo private similar to other GW code?